app/models/videogame.rb

```diff
@@ -1,5 +1,3 @@
 class Videogame < ApplicationRecord
   validates :name, presence: true
-
-  has_many :reviews
```

Comment:
actually this would be the correct time to update the association as we have added the `belongs_to :videogame` in `review.rb`